lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -481,6 +481,14 @@ public function getDropViewSQL($name)
     }
 
     /**
+     * {@inheritDoc}
+     */
+    public function getRandomFunctionSQL()
+    {
+        return 'DBMS_RANDOM.VALUE';
```

Comment:
Documentation states it produces a number like this:

`DBMS_RANDOM.VALUE produces numbers in [0,1) with 38 digits of precision.`

Not really inteneded, is it?